packages/docgen/src/index.js

```diff
@@ -113,9 +124,9 @@ module.exports = function( sourceFile, options ) {
 	}
 
 	if ( options.formatter ) {
-		runCustomFormatter( path.join( processDir, options.formatter ), processDir, doc, filteredIr, 'API' );
+		runCustomFormatter( path.join( processDir, options.formatter ), processDir, doc, filteredIR, 'API' );
```

Comment:
There's a few other violations in the module, but I'd tried to keep the changes limited.

>Acronyms must be written with each of its composing letters capitalized. This is intended to reflect that each letter of the acronym is a proper word in its expanded form.

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#abbreviations-and-acronyms

Reply:
:+1: